src/views/virtualmachines/details/tabs/overview/components/VirtualMachinesOverviewTabUtilization/components/NetworkUtil/NetworkUtil.tsx

```diff
@@ -20,38 +24,91 @@ const NetworkUtil: React.FC<NetworkUtilProps> = ({ vmi }) => {
     endTime: currentTime,
   });
 
+  const [networkTotal] = usePrometheusPoll({
+    query: queries?.NETWORK_TOTAL_BY_INTERFACE_USAGE,
+    endpoint: PrometheusEndpoint?.QUERY,
+    namespace: vmi?.metadata?.namespace,
+    endTime: currentTime,
+  });
+
   const [networkOut] = usePrometheusPoll({
     query: queries?.NETWORK_OUT_USAGE,
     endpoint: PrometheusEndpoint?.QUERY,
     namespace: vmi?.metadata?.namespace,
     endTime: currentTime,
   });
-
+  const networkInterfaceTotal = Number(networkTotal?.data?.result?.[0]?.value?.[1])?.toFixed(2);
   const networkInData = +networkIn?.data?.result?.[0]?.value?.[1];
   const networkOutData = +networkOut?.data?.result?.[0]?.value?.[1];
-
   const totalTransferred = xbytes(networkInData + networkOutData || 0, {
     iec: true,
     fixed: 0,
   });
-
   const isReady = !isEmpty(networkInData) || !isEmpty(networkOutData);
 
   return (
     <div className="util network">
       <div className="util-upper">
         <div className="util-title">
           {t('Network Transfer')}
-          <div className="util-title__subtitle text-muted">{t('Primary Network')}</div>
+          <Popover
+            bodyContent={
+              <div>
+                <Text component={TextVariants.h3}>{t('Network transfer breakdown')}</Text>
+                <Text component={TextVariants.h6}>{t('Top consumer')}</Text>
+                {interfacesNames?.map((networkInterface) => (
+                  <div key={networkInterface?.name} className="network-popover">
+                    <Link
+                      to={`${getResourceUrl({
+                        model: VirtualMachineModel,
+                        resource: vmi,
+                      })}/metrics?network=${networkInterface?.name}`}
+                    >
+                      {networkInterface?.name}
+                    </Link>
+                    {networkInterface?.name === networkInterfaceTotal ? (
+                      <div className="text-muted">{`${networkInterfaceTotal} MBps`}</div>
+                    ) : (
+                      <div className="text-muted">
+                        {networkTotal?.data?.result?.map(
+                          (name) =>
+                            name?.metric?.interface === networkInterface?.name &&
+                            `${Number(name?.value?.[1])?.toFixed(2)} MBps`,
+                        )}
+                      </div>
+                    )}
+                  </div>
+                ))}
+                {interfacesNames?.length > 5 && (
+                  <Link
+                    to={`${getResourceUrl({
+                      model: VirtualMachineModel,
+                      resource: vmi,
+                    })}/metrics?network`}
+                  >
+                    {t('View more')}
+                  </Link>
+                )}
+              </div>
+            }
+            position="bottom"
+          >
+            <div>
+              <Button variant="link" isInline>
```

Comment:
It's better to use `variant={ButtonVariant.link}` :) 
(Sorry, I didn't see that earlier)

Reply:
Thank you Hilda ! :)